src/onboarding/Create/Create.js

```diff
@@ -509,6 +511,33 @@ const Create = React.memo(function Create({
     [web3]
   )
 
+  const applyEstimatePriorityFee = useCallback(
+    async transaction => {
+      const priorityFeeHistory = await web3.eth.getFeeHistory('4', 'latest', [
+        10,
+      ])
+      if (
+        priorityFeeHistory &&
+        priorityFeeHistory.reward &&
+        priorityFeeHistory.reward.length > 0
+      ) {
+        // takes the top 10 of the last 4 blocks and take the average after removing zero values
+        const feeHistories = priorityFeeHistory.reward
+          .map(fee => web3.utils.hexToNumber(fee[0]))
+          .filter(fee => fee > 0)
+        return {
+          ...transaction,
+          maxPriorityFeePerGas: Math.round(
+            feeHistories.reduce((acc, fee) => acc + fee, 0) /
+              feeHistories.length
```

Comment:
Beware, there is a chance that filtering `priorityFeeHistory` may return an empty array. In this case `feeHistory.length === 0` and you'd be deviding by zero (Which would give you `Infinity` because... Javascript reasons...)

Reply:
even tho the RPC should never return only 0s I added a check for it

Reply:
Hi @WissenIstNacht, Aragon dTech chap here, 🤞 you find time for this, as customers are on our neck with not being able to create DAOs on Poly 😅